core/src/main/java/de/danoeh/antennapod/core/feed/FeedItemFilter.java

```diff
@@ -56,9 +66,17 @@ public FeedItemFilter(String[] properties) {
                 case "has_media":
                     showHasMedia = true;
                     break;
+                case "no_media":
+                    showNoMedia = true;
+                    break;
                 case "is_favorite":
                     showIsFavorite = true;
                     break;
+                case "not_favorite":
+                    showNotFavorite = true;
+                    break;
+                default:
+                    return;
```

Comment:
This means that the filter will stop completely and silently if there is an unknown value. I think it would be better to just ignore that value by calling `break` instead.

Reply:
Okay  :+1: 
There should be no unknown values now.